picard/ui/options/plugins.py

```diff
@@ -230,6 +234,22 @@ def plugin_updated(self, plugin_name):
                 self.change_details()
                 break
 
+    def plugin_uninstall(self):
+            plugin = self.items[self.ui.plugins.selectedItems()[0]]
+            buttonReply = QtWidgets.QMessageBox.question(self, 'Confirmation',
+                    "Do you really want to uninstall the plugin %s ?" % plugin.name,
```

Comment:
Add quotes around plugin's name as you did below:

```python
 "Do you really want to uninstall the plugin '%s' ?" % plugin.name,
```

Also it is require to enclose the string with `_()` for it to be translatable.